app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainFragment.kt

```diff
@@ -151,10 +151,10 @@ class MainFragment : Fragment() {
 
         fun setCountryListViewData(caseData: Cases) {
             val casesListViewItems = ArrayList<Cases>()
-            for(x in 5 until caseData.country_name.size) {
+            for(x in 5 until caseData.countryName.size) {
                 casesListViewItems.add(Cases(cases = arrayListOf(caseData.cases[x]), deaths = arrayListOf(caseData.deaths[x]),
                     totalRecovered = arrayListOf(caseData.totalRecovered[x]), activeCases = arrayListOf(caseData.activeCases[x]),
-                    newCases = arrayListOf(caseData.newCases[x]), country_name = arrayListOf(caseData.country_name[x]), newDeaths = arrayListOf(caseData.newDeaths[x]),
```

Comment:
Changed to better fit Kotlin naming conventions https://kotlinlang.org/docs/reference/coding-conventions.html